config.json

```diff
@@ -398,7 +398,12 @@
       "core": false,
       "difficulty": 6,
       "slug": "food-chain",
-      "topics": null,
+      "topics": [
+ +      	"arrays",
```

Comment:
These `+`s shouldn't be here, that's what's making the build fail :) Apart from that I think you've got a good list of suggested topics! :) The only topic I'm not sure about is `transforming`. How would you say that's related to this exercise?

Reply:
Oh goodness, I can't believe I missed that. haha

Thank you so much! :)